tests/test_features/test_pca.py

```diff
@@ -0,0 +1,71 @@
+##########################################################################
+## Imports
+##########################################################################
+
+import unittest
+import numpy as np
+import numpy.testing as npt
+
+from tests.dataset import DatasetMixin
+from yellowbrick.features.pca import PCA2D
+
+##########################################################################
+## RadViz Base Tests
+##########################################################################
+
+class PCA2DTests(unittest.TestCase, DatasetMixin):
+
+    def test_scale_t_center_t(self, scale=True, center=True):
```

Comment:
Our test package requires a one line description of the tests, e.g. 

```python
def test_scale_t_center_t(self):
    """
    Test the PCA2D visualizer scaled and centered 
    """
```

Because this line in the docstring is printed in the test report. 

Also, in terms of arguments to the test, e.g. (scale=True, center=True), our test runner won't be able to modify the arguments, so they're there for documentation only, and I think the name of the method does a good job of documenting it if you want to remove them. Otherwise you could create a function that takes the arguments that is not named `test_` and then simply pass the arguments to the function from your two test cases.